### PR TITLE
docs: Fix type URI for custom_response example

### PR DESCRIPTION
**Pull request by pchaseh**
Commit Message: docs: Fix type URI for custom_response example
Additional Description: The type URI `envoy.extensions.filters.http.custom_response.v3.LocalResponsePolicy` is not/no longer valid, the correct URI is `envoy.extensions.http.custom_response.local_response_policy.v3.LocalResponsePolicy` as evidenced by the fact that there are no longer any other references to the former for all of Envoy's codebase
Risk Level: Low
Testing:
Docs Changes: The use of `envoy.extensions.filters.http.custom_response.v3.LocalResponsePolicy` in the custom_response example is updated with `envoy.extensions.http.custom_response.local_response_policy.v3.LocalResponsePolicy`
Release Notes: N/A
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Review by pradeepcrao — COMMENTED** [review]
Thank you so much for the fix!

File: api/envoy/extensions/filters/http/custom_response/v3/custom_response.proto [hunk 1]
```diff
@@ -48,7 +48,7 @@ message CustomResponse {
   //         action:
   //           name: action
   //           typed_config:
-  //             "@type": type.googleapis.com/envoy.extensions.filters.http.custom_response.v3.LocalResponsePolicy
+  //             "@type": type.googleapis.com/envoy.extensions.http.custom_response.local_response_policy.v3.LocalResponsePolicy
```
[the remark this answers is not in the document]
Comment (pchaseh) [reply api/envoy/extensions/filters/http/custom_response/v3/custom_response.proto:51]: Updated

**Review by adisuissa — APPROVED** [review]
LGTM, thanks for fixing this!
/lgtm api

**Comment by adisuissa** [comment]
Oops, please fix DCO

**Review by adisuissa — APPROVED** [review]
/lgtm api
LGTM, thanks!

**Pull request merged**